WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert Student Grading

## Changes

Converted Student Grading

## Test

1. Open the Classroom Monitor for a run
2. Click on "Grade By Team"
3. Click on a workgroup
4. Make sure everything looks and works like before
- Student work shows up
- Student scores show up
- Scores update immediately when teacher changes them
- Steps expand
- Steps collapse
- Sorting works
- Changing period works
- The UI looks good when the browser has narrow width

Closes #887
